raster/r.what/testsuite/test_r_what.py

```diff
@@ -78,6 +79,356 @@ class TestRasterWhat(TestCase):
 511235.1561495|135372.358883875||7
 332533.5941495|242831.139883875||121
 """
+    reference_points_json = [
```

Comment:
You could create `reference_points_json` from the `reference_points` string.

Reply:
Will do.

Reply:
Done.